layers/+lang/javascript/packages.el

```diff
@@ -19,6 +19,7 @@
     flycheck
     ggtags
     helm-gtags
+    indium
```

Comment:
You need to document the additional repl choice in the README. In addition you need to extend the documentation for the layer variables to show that the new setting of `indium` is supported.